src/vde_backend/pyToCassandra.py

```diff
@@ -147,7 +146,6 @@ def createTable(session, keyspace, table_name, columns, primary_keys, clustering
 	query += "{})) ".format(',' + ','.join(clustering_keys) if len(clustering_keys) else '')
 	query += "{};".format(getOrdering(ordering))
 
-	logging.debug("===>  create table query : " + query)

```

Comment:
I find the logging of queries very useful for debugging purposes. Why removing them?

Reply:
Please keep it because it is very helpful for debugging. If the logs are too cluttered for your taste, you can filter them using `less`. See https://stackoverflow.com/a/11329255/8289769